src/main/java/Tyrone.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Scanner;
+
+import tyrone.constants.Constants;
+import tyrone.exceptions.MissingTimeInfoException;
+import tyrone.exceptions.TyroneException;
+import tyrone.exceptions.WrongDeadlineFormatException;
+import tyrone.exceptions.WrongEventFormatException;
+import tyrone.task.Deadline;
+import tyrone.task.Event;
+import tyrone.task.Task;
+import tyrone.task.ToDo;
+
+public class Tyrone {
+    public static void getUserInput(String userInput) throws TyroneException {
```

Comment:
This method is very long, you may consider extracting each case into separate methods. It will be better to keep methods short.